certified-connectors/Ephesoft/apiDefinition.swagger.json

```diff
@@ -0,0 +1,356 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Ephesoft Semantik For Invoices",
+    "description": "The Ephesoft Semantik for Invoices connector helps organizations extract critical invoice data using cutting edge AI models to automate accounts payable processing.  Semantik Invoice is a point solution that can speed up your invoice processing, reduce compliance errors, and free up your staff to work on more business critical items.",
+    "version": "1.0",
+    "contact": {
+      "name": "Ephesoft Semantik Support",
+      "url": "https://intercom.help/ephesoft/en",
+      "email": "tickets@ephesoft.com"
+    }
+  },
+  "host": "api.us.ephesoft.io",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://ephesoft.com/products/semantik-invoice/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://ephesoft.com/privacy-policy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Finance;Productivity"
+    }
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/settings/integrations/configurations": {
+      "x-ms-notification-content": {
+        "description": "Webhook Response",
+        "schema": {
+          "$ref": "#/definitions/WebhookResponse"
+        },
+        "parameters": {},
+        "responses": {}
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "201",
```

Comment:
@Ephesoft-Stitus Please provide a valid text description instead of the response code.

Reply:
@sriyen-msft - Good catch, updated description to 'Created'.